teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -658,6 +662,8 @@ enum class TestStatus {
 
 @Deprecated(message = "Deprecated due to unclear naming. use TestRun class", replaceWith = ReplaceWith("TestRun"))
 interface TestOccurrence {
+    val id : TestOccurenceId
```

Comment:
Why do you add a property to a deprecated class? And add a new class which names is derived from name which is described as unclear in the comment?